cli/cliui/provisionerjob.go

```diff
@@ -41,6 +41,17 @@ type ProvisionerJobOptions struct {
 	Silent bool
 }
 
+type ProvisionerJobError struct {
+	Message string
+	Code    codersdk.JobErrorCode
+}
+
+var _ error = new(ProvisionerJobError)
+
+func (err *ProvisionerJobError) Error() string {
+	return err.Message
```

Comment:
Just an idea, could be useful sometime.

```suggestion
	return fmt.Sprintf("%s (%s)", err.Message, err.Code)
```

Reply:
Thanks for suggestion, but this message will be shown in CLI or site, let's not scare the customer :)